.github/workflows/testing-integration.yaml

```diff
@@ -3,6 +3,54 @@ name: "Integration Tests"
   workflow_call: {}
 
 jobs:
+  # setup-index:
```

Comment:
Leaving this commented code in as I have plans to bring back CI integration tests in a future PR.